lib/ex_doc/cli.ex

```diff
@@ -35,6 +36,11 @@ defmodule ExDoc.CLI do
     else
       generate(args, opts, generator)
     end
+
+    if List.keymember?(opts, :warnings_as_errors, 0) and ExDoc.WarningCounter.count() > 0 do
```

Comment:
I think it is clearer in intention to use `Keyword.has_key?/2` since opts is a keyword.

```suggestion
    if Keyword.has_key?(opts, :warnings_as_errors) and ExDoc.WarningCounter.count() > 0 do
```